api-extractor-esm.json

```diff
@@ -0,0 +1,4 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../common/build/build-common/api-extractor-base-esm.json"
```

Comment:
How does this work exactly? The relative path can't be correct. (Is it not used and is a template? If so, can we have a place for such things?)

Reply:
This file shouldn't be there. I'll delete it. It's leftover from when I used Pnpm exec to copy the file into all the packages.